crypto/fipsmodule/sha/sha3_test.cc

```diff
@@ -43,13 +63,30 @@ class SHA3TestVector {
     uint8_t digest[SHA3_256_DIGEST_LENGTH];
     EVP_MD_CTX* ctx = EVP_MD_CTX_new();
     
-    // Test the correctness via the single shot API.
+    // Test SHA3 when |experimental_unstable_enable_sha3| is disabled
+    // |experimental_unstable_enable_sha3| is desabled by default
+    // SHA3 should fail when |experimental_unstable_enable_sha3| is disabled
+    if (*experimental_unstable_enable_sha3_get() == 0){
```

Comment:
A user can run the test suite, but not really interact or write code for it without modifying the source code. Because what is being run is an actual executable.